lib/ringcaptcha/api_stub.rb

```diff
@@ -1,9 +1,20 @@
 require 'json'
 require 'ringcaptcha/response'
+require 'ringcaptcha/instrumentation'
 
 module Ringcaptcha
   module APIStub
     def self.call(api_key, app_key, path, params = {})
+      if Ringcaptcha.use_instrumentation
```

Comment:
This feels like asking instead of telling. Also this method eventually calls `call_ringcaptcha`, and the conditional here makes it feel like it does 2 separate things according to the value of `use_instrumentation`. It doesn't.

How about instead you call a block, something like

```
with_instrumentation do
  call_ringcaptcha(api_key, app_key, path, params = {})
end
```

`with_instrumentation` will either instrument the block (if required), or simply yield it. WDYT?